app/javascript/packages/document-capture/components/acuant-capture-canvas.jsx

```diff
@@ -183,9 +271,28 @@ function AcuantCaptureCanvas({
     [onImageCaptureSuccess, onImageCaptureFailure],
   );
   const [captureType, setCaptureType] = useState(/** @type {AcuantCaptureType} */ ('AUTO'));
+  const [frameState, setFrameState] = useState(
+    /** @type {AcuantFrameState} */ (AcuantDocumentState.NO_DOCUMENT),
+  );
+  const captureStatus = useMemo(() => getCaptureStatus(captureType, frameState), [
+    captureType,
+    frameState,
+  ]);
 
   useEffect(() => {
+    let originalAcuantCameraStart;
     if (isReady) {
+      originalAcuantCameraStart = /** @type {AcuantGlobal} */ (window).AcuantCamera.start;
```

Comment:
I hoped to be able to use Acuant's `onFrameAvailable` method for this:

https://github.com/Acuant/JavascriptWebSDKV11/tree/master/SimpleHTMLApp#acuantcameraui

Despite what the documentation implies, it doesn't appear that it's always called when resetting back to "Align" (no document found) from "Move Closer" or "Capturing" states.

You can see this in [the source](https://raw.githubusercontent.com/Acuant/JavascriptWebSDKV11/master/SimpleHTMLApp/webSdk/dist/AcuantJavascriptWebSdk.js) starting on line 110, that the "Align" state (represented by empty `response`) prevents the call to `onFrameAvailable`:

```js
  function handleLiveCapture(response, captureCb){
    if (response) {
      if (onDetectedResult && onDetectedResult.state === -1) {
        return;
      }
      else {
        if(captureCb.onFrameAvailable){
          captureCb.onFrameAvailable(response);
        }
```

And at line 305, absence of result treated as "Align":

```js
  function handleUi() {
    if (!onDetectedResult) {
      drawPoints("#000000");
      drawText(userOptions.text.NONE);
    }
```